### DISCUSSION TITLE
Add KelvinPerMinute to TemperatureChangeRateUnit?

### DISCUSSION THREAD

**Discussion by andrewmeservy** [angularsen/UnitsNet#1428] [Ideas]
Would it be possible to add KelvinPerMinute as a TemperatureChangeRateUnit?

I saw on the guide that some quantities/units may not be a good fit to add, so I thought I would ask before I tried to follow the instructions to add it.

**Comment by angularsen** [comment]
Thanks for asking, yes, sounds good to me 👍 The bar is lower for adding units to existing quantities, usually OK.

**Comment by andrewmeservy** [comment]
I managed to add this today, thank you for having such good documentation! It made the process super easy.
PR: https://github.com/angularsen/UnitsNet/pull/1429
